code/modules/status_bar/status_bar.dm

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+/client/var/status_bar_prev_text = ""
```

Comment:
I know it's modular and stuff (and personally I like it), but keep this with client_defines.dm. Otherwise, it fucks with SpacemanDMM, and is also inconsistent.

Reply:
> fucks with SpacemanDMM

how

Reply:
It used to be with turfs that when you ctrl+click `turf/` it takes you to the very slim definition in the LINDA files that just has like, the gas or something. It just picks whichever one it can, not the most complete one.